app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -16,55 +16,53 @@
 
-    private ArrayAdapter arrayAdapter;
+    private ArrayAdapter mAdapterApps;
 
-    private ListView listViewApplications;
+    private ListView mListViewApps;
 
-    private ApplicationDao applicationDao;
+    private ApplicationDao mApplicationDao;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        Timber.i("onCreate");
+//        Timber.i("onCreate");
         super.onCreate(savedInstanceState);
-
         setContentView(R.layout.activity_app_list);
 
-        listViewApplications = findViewById(R.id.listViewApplications);
-
-        textViewLastSynchronization = findViewById(R.id.textViewLastSynchronization);
-
         BaseApplication baseApplication = (BaseApplication) getApplication();
-        applicationDao = baseApplication.getDaoSession().getApplicationDao();
+        mApplicationDao = baseApplication.getDaoSession().getApplicationDao();
+
+        initViews();
     }
 
-    @Override
-    protected void onStart() {
-        Timber.i("onStart");
-        super.onStart();
+    private void initViews() {
+        mListViewApps = findViewById(R.id.listViewApplications);
 
+        mTextViewLastSynchronization = findViewById(R.id.textViewLastSynchronization);
         // Display the time of last synchronization with the server
-        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
-        long timeOfLastSynchronization = sharedPreferences.getLong(AppSynchronizationActivity.DownloadAppListAsyncTask.PREF_LAST_SYNCHRONIZATION, 0);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences
+                (getApplicationContext());
+        long timeOfLastSynchronization = sharedPreferences.getLong(AppSynchronizationActivity
+                .DownloadAppListAsyncTask.PREF_LAST_SYNCHRONIZATION, 0);
         Date date = new Date(timeOfLastSynchronization);
-        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm");
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(Const.DATE_FORMAT_LAST_SYNC);
         String dateAsString = simpleDateFormat.format(date);
-        String lastSynchronization = getString(R.string.last_synchronization);
-        textViewLastSynchronization.setText(lastSynchronization + ": " + dateAsString);
+        mTextViewLastSynchronization.setText(String.format(getString(R.string
+                .last_synchronization), dateAsString));
 
         // Load the list of Applications stored in the local database
-        applications = applicationDao.loadAll();
-        Timber.i("applications.size(): " + applications.size());
-        arrayAdapter = new AppListArrayAdapter(getBaseContext(), R.layout.activity_app_list_item, applications);
-        listViewApplications.setAdapter(arrayAdapter);
-
-
+        mApplications = mApplicationDao.loadAll();
+//        Timber.i("mApplications.size(): " + mApplications.size());
```

Comment:
Don't disable log output. This makes debugging and looking for causes of errors harder in the future.